test/src/measure_iperf.py

```diff
@@ -0,0 +1,80 @@
+import autotest as autotest
+from configparser import ConfigParser
+from argparse import (ArgumentParser, ArgumentDefaultsHelpFormatter, Namespace,
+                      FileType, ArgumentTypeError)
+from argcomplete import autocomplete
+from logging import (info, debug, error, warning,
+                     DEBUG, INFO, WARN, ERROR)
+from server import Host, Guest, LoadGen, MultiHost
+from enums import Machine, Interface, Reflector
+from measure import AbstractBenchTest, Measurement, end_foreach, BRIEF, OUT_DIR
+from util import safe_cast, product_dict
+from typing import Iterator, cast, List, Dict, Callable, Tuple, Any
+import time
+from os.path import isfile, join as path_join
+import yaml
+import json
+from root import *
+from dataclasses import dataclass, field
+import subprocess
+
+
+
+@dataclass
+class IPerfTest:
+
+    direction = "forward" # can be: forward | reverse | bidirectional
+
+    # network options - currently set up to work with the "iperf_test.cfg" config
+    guest_test_iface = "eth1"
+    guest_hostname = "10.2.0.1"
+    loadgen_hostname = "10.2.0.2"
+    subnet_size = 24
+    port = 5001
```

Comment:
these settings shouldnt be here, but instead only in the respective test/conf/*.cfg